librocksdb-sys/build.rs

```diff
@@ -103,14 +103,24 @@ fn build_rocksdb() {
         // This is needed to enable hardware CRC32C. Technically, SSE 4.2 is
         // only available since Intel Nehalem (about 2010) and AMD Bulldozer
         // (about 2011).
-        config.define("HAVE_SSE42", Some("1"));
-        config.flag_if_supported("-msse2");
-        config.flag_if_supported("-msse4.1");
-        config.flag_if_supported("-msse4.2");
+        let target_feature = env::var("CARGO_CFG_TARGET_FEATURE").unwrap();
+        let target_features: Vec<_> = target_feature.split(",").collect();
+        if target_features.contains(&"sse2") {
```

Comment:
Does it change the behavior to not enable sse2 etc by default? If that's the case, CHANGELOG.md should be updated accordingly so developers are aware about the performance degradation once they upgrade the rocksdb crate.

Reply:
I believe SSE2 is in the set of default features for x86-64, at least it is for me if I run
```sh

Reply:
Maybe something like "RocksDB is not compiled with SSE4 instructions anymore unless the corresponding features are enabled in rustc"?